### PR TITLE
USHIFT-1526: Update kubelet config during rebase

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
/retest
```
Error: Timed out waiting for pod image(s) from the 'openshift-storage' namespace to be downloaded
```

File: assets/core/kubelet.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
```
[the remark this answers is not in the document]
Comment (pmtk) [reply assets/core/kubelet.yaml:2]: Yes, our new kubelet config is recreated from upstream's https://github.com/openshift/machine-config-operator/blob/master/templates/master/01-master-kubelet/_base/files/kubelet.yaml#L6

File: pkg/node/kubelet.go [hunk 2]
```diff
@@ -92,50 +95,40 @@ func (s *KubeletServer) writeConfig(cfg *config.Config) error {
...
+	var data bytes.Buffer
+	if err := tpl.Execute(&data, tplParams); err != nil {
+		return fmt.Errorf("templating kubelet config failed: %w", err)
+	}
 
 	// Load real resolv.conf in case systemd-resolved is used
 	// https://github.com/coredns/coredns/blob/master/plugin/loop/README.md#troubleshooting-loops-in-kubernetes-clusters
 	if _, err := os.Stat(config.DefaultSystemdResolvedFile); err == nil {
-		data = append(data, fmt.Sprintf("\nresolvConf: %s\n", config.DefaultSystemdResolvedFile)...)
+		data.WriteString(fmt.Sprintf("\nresolvConf: %s\n", config.DefaultSystemdResolvedFile))
```
[the remark this answers is not in the document]
Comment (pmtk) [reply pkg/node/kubelet.go]: Yes, PTAL at newest commit

File: assets/core/kubelet.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhellmann) [reply assets/core/kubelet.yaml:2]: ack, ok

**Pull request merged**